src/components/Header.tsx

```diff
@@ -6,13 +6,18 @@ const Header = () => (
       <h1 className="header-text">Welcome to the </h1>
       <Image
         src="/lola-logo.svg"
-        className="header-content"
+        className="header-image"
         alt="Lola logo"
         width={266}
         height={50}
       />
       <h1 className="header-text">Staff Handbook</h1>
     </div>
+    <div className="header-button">
+      <a href="admin.html" className="cms-button">
```

Comment:
A tiny detail... This will work just fine right now, but if this header ever got used in a page which isn't at the root of the site this relative url would fail.

```suggestion
      <a href="/admin.html" className="cms-button">
```

Reply:
Got it, thank you for the tip!